octopoes/tools/rename-origin-method.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import json
+import logging
+
+import click
+from xtdb_client import XTDBClient
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(
+    context_settings={
+        "help_option_names": ["-h", "--help"],
+        "max_content_width": 120,
+        "show_default": True,
+    }
+)
+@click.option("-n", "--node", default="0", help="XTDB node")
+@click.option(
+    "-u",
+    "--url",
+    default="http://localhost:3000",
+    help="XTDB server base url",
+)
+@click.option(
+    "-t",
+    "--timeout",
+    type=int,
+    default=5000,
+    help="XTDB request timeout (in ms)",
+)
+@click.option("-v", "--verbosity", count=True, help="Increase the verbosity level")
+@click.pass_context
+def cli(ctx: click.Context, url: str, node: str, timeout: int, verbosity: int):
+    verbosities = [logging.WARN, logging.INFO, logging.DEBUG]
+    try:
+        if verbosity:
+            logging.basicConfig(level=verbosities[verbosity - 1])
+    except IndexError:
+        raise click.UsageError("Invalid verbosity level (use -v, -vv, or -vvv)")
+
+    client = XTDBClient(url, node, timeout)
+    logger.info("Instantiated XTDB client with endpoint %s for node %s", url, node)
+
+    ctx.ensure_object(dict)
+    ctx.obj["client"] = client
+
+
+@cli.command(help="List observation origins based on method")
+@click.argument("method", required=False)
+@click.pass_context
+def list(ctx: click.Context, method: str):  # noqa: A001
+    if method:
+        query = f'{{:query {{:find [(pull ?var [*])] :where [[?var :type "Origin"][?var :origin_type "observation"][?var :method "{method}"]]}}}}'  # noqa: E501
+    else:
+        query = '{:query {:find [(pull ?var [*])] :where [[?var :type "Origin"][?var :origin_type "observation"][?var :method]]}}'  # noqa: E501
```

Comment:
There is no need to ignore E501, the query string can be a multi-line string. The query can also be written in a more readable way by indenting the nested clauses when using a multi-line string. This would also be consistent with how we write the queries in other places, see ooi_repository.py for example.

Reply:
The first thing I always when fetching queries for files I have not writ is make a one-liner of the query. But if this is specific KAT policy I shall yield.

Reply:
Addressed in aac161f3852157b38ea74b99e609785434ec47da